sql/2020/19_Compression/mbyte_saving_of_text_compression.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+#Text Based Compression Byte Savings
+SELECT 
+  _TABLE_SUFFIX AS client,
+  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.details.overallSavingsBytes")as INT64)/1024/1024) mbyte_savings,
```

Comment:
Is MB too coarse or should this be KB, for example 100 KB increments?

Additional formatting suggestions:
```suggestion
  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.details.overallSavingsBytes") AS INT64) / 1024 / 1024) AS mbyte_savings,
```

Reply:
like the idea of 100KB increments, and thought as seen of which some of [httparachive reports](https://httparchive.org/reports/loading-speed) are intercepted by p10/25/50/75/90 might be a good try on. then made some change on the file. please have a look. thanks.